docs/version0.44.md

```diff
@@ -76,6 +77,17 @@ The `jcmd` tool also now supports specifying `0` as a VMID to target all VMs.
 
 For more information, see [Java diagnostic command (`jcmd`) tool](tool_jcmd.md).
 
+### Field names in `J9BuildFlags` changed
+
+The Direct Dump Reader (DDR) code enables reading system dump data by using the OpenJ9
+Diagnostic Tool Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool.
+DDR code uses fields of `J9BuildFlags` to access build flags in the system dump data.
+The names of `J9BuildFlags` fields changed over time because of which supporting different system dumps with different build flags became a challenge.
```

Comment:
This sentence feels awkward; perhaps some punctuation will help and maybe remove the first occurrence of "different"?
What was your objection to my suggestion "making it challenging to support all system dumps"?

Reply:
Even though I understand the sentence but the problem was with "... over time making it challenging...".  a noun followed by an "ing" as per Acrolinx. There is a possibility that the translation will find this confusing... is it "time making"? What is "it"? Also, leaving the sentence with "all system dumps". It was not clear what was challenging.

I will delete the first instance of "different". I hope that is ok.

Reply:
I expect punctuation (a comma) would have been helpful in my suggestion
> fields changed over time, making it challenging ...

I think it would be a similar improvement to your proposal
> fields changed over time, because of which ...

In both cases it is the "change" that makes it challenging, or it is because of the "change". I don't how we've avoided the problem you describe.

Reply:
How about "The names of `J9BuildFlags` fields changed over time and therefore, supporting system dumps with different build flags became a challenge."?

Reply:
Also, because of Peter's suggestion about mentioning DDR code in the heading, I am also adding a sentence at the end of the second para.

> Earlier, field names in `J9BuildFlags` were based on names defined in `j9.flags`. Now, when the `J9BuildFlags` is generated for each build, the flag names are those names that are specified in `j9cfg.h` (derived from `j9cfg.h.ftl` or `j9cfg.h.in`) instead of the names that are defined in `j9.flags`. For example, `env_data64` is now referred to as `J9VM_ENV_DATA64`. Because of this change, reference to the field names in `J9BuildFlags` is changed in the DDR code as well.

Is that ok?